packages/vitetris.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Vitetris < Package
+  description 'Classic multiplayer tetris for the terminal'
+  homepage 'https://www.victornils.net/tetris/'
+  version '0.59.1'
+  license 'BSD-2-Clause'
+  compatibility 'all'
+  source_url 'https://github.com/vicgeralds/vitetris.git'
+  git_hashtag "master"
+
+  depends_on "curl"
+  depends_on "llvm"
+  depends_on "make"
+
+  def self.build
+    system "git checkout 9bbea4f5ab35d6fb8fe70c14daba19e7f35ab44e"
+    system 'make -s CFLAGS="-O2 -pipe -Wno-error=implicit-int -Wno-error=implicit-function-declaration"'
```

Comment:
Take a look at how we format additional CFLAGS elsewhere in the project, and provide a rationale for the `-Wno-error` ones.

Reply:
You will need to use `CREW_PREFIX` to abstract the install location.

Reply:
The specific makefile will have its own way of setting prefixes, look at our other code to see how it usually goes.

> You will need to use `--prefix=#{CREW_PREFIX}` to abstract the install location.

That's for configure, and we abstract that away with autotools now.